thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -273,7 +272,7 @@ class CIndexReadSlaveBase : public CSlaveActivity
                                     continue; // try next copy and ultimately failover to local when no more copies
                                 }
                                 ActPrintLog("[part=%d]: reading remote dafilesrv index '%s' (logical file = %s)", partNum, path.str(), logicalFilename.get());
-                                partNum = p;
+                                partNum = (p+1);
```

Comment:
Change Relates to: https://github.com/hpcc-systems/HPCC-Platform/pull/18144/commits/55ef2b94c7d452dbec6d168ce3e3f9cee8cca2c4#r1426670630